src/Auth/Models/User.php

```diff
@@ -25,8 +25,8 @@ class User extends Model implements \Illuminate\Contracts\Auth\Authenticatable
      */
     public $rules = [
         'email' => 'required|between:3,255|email|unique:users',
-        'password' => 'required:create|min:2|confirmed',
-        'password_confirmation' => 'required_with:password|min:2'
+        'password' => 'required:create|min:4|confirmed',
+        'password_confirmation' => 'required_with:password|min:4'
```

Comment:
@LukeTowers I'm not quite sure the reasoning that @daftspunk changed this back to `2` (I presume for backwards compatibility), but if we are going to change it, I personally would rather this be `min:6` or `min:8` at the least. We should be following best practices for password selection, and 4 character passwords are too easy to brute-force in this day and age. (I understand we have throttling, but it can be disabled).

Reply:
@bennothommo I checked the git history and the value was originally 4, so I don't know why @daftspunk felt the need to bump it down to 2. I would also prefer the minimum be set to 8, but I believe we had a conversation about this a while ago and it was decided that people could always just bump up the rules themselves. Additionally, this is somewhat tied into the autogeneration of admin passwords PR, was that one ever finished?

Reply:
@LukeTowers I do recall that conversation. On further consideration though, I feel if anything we should be bumping it high and forcing people to work to make the value lower, so that they are fully responsible for the consequences.

Re: auto-generation of admin passwords, it's more or less [finished](https://github.com/octobercms/october/pull/4866), but on hold until after L6.

Reply:
I'll have another convo with @daftspunk about it and get back to you

Reply:
Spoke to Sam and he re-iterated that
> it's best to leave it as 4 otherwise it's a breaking change. Password policies are a complex topic and so it's best not to have any strong opinions and leave it up to a plugin or attend to the entire problem holistically
> there are legit applications that use short passwords
> Namely demo apps and temporary passwords
> A config item would probably appease whatever is driving this. Default without a breakage

So we could add a new config item in `auth.minPasswordLength` that defaults to `8` in the file but in the Config::get() it defaults to the existing 4. That way existing installs won't break, and new ones will default to 8.

Also we might want to look at namespacing the auth config under auth.backend instead to better support any future changes bringing us closer to Laravel's auth system.